docs/manual/config/keys.rst

```diff
@@ -45,6 +45,82 @@ The :class:`EzKey` modifier keys (i.e. ``MASC``) can be overwritten through the
        'C': 'control',
     }
 
+KeyChords
+=========
+
+Qtile also allows sequences of keys to trigger callbacks. In Qtile, these
+sequences are known as chords and are defined with
+:class:`libqtile.config.KeyChord`. Chords are added to the ``keys`` section of
+the config file.
+
+::
+
+    from libqtile.config import Key, KeyChord
+
+    keys = [
+        KeyChord([mod], "z", [
+            Key([], "x", lazy.spawn("xterm"))
+        ])
+    ]
+
+The above code will launch xterm when the user presses Mod + z, followed by x.
+
+.. warning::
+    Users should note that key chords are aborted by pressing <escape>. In the
+    above example, if the user presses Mod + z, any following key presses will
+    still be sent to the currently focussed window. If <escape> has not been
+    pressed, the next press of x will launch xterm.
```

Comment:
So with this configuration, if I hit `mod+z` and then `y`, the mode remains until `escape` or `x` is hit, passing anything else through?

Reply:
Commenting again after reading the modes bit below. What I understood from the description is that in the above (without a `mode`), when the first key inside the mode is hit, the mode is exited. And below (with a `mode`) the mode remains until `escape` is hit, even if you hit some of the nested key bindings. Is that right?

Reply:
Yes. See [this thread](https://groups.google.com/u/1/g/qtile-dev/c/r87i-fdIW2o/m/dpy8bowIAQAJ) on the qtile-dev group for more.

I might try and take a look at fixing the bug later but, for now, I've put the warning in the docs.

Reply:
Just say your second comment (we posted at the same time). 

With modes, once you enter the mode, you stay in it until you leave by pressing escape. While in the mode, you can then trigger subcommands numerous times.

Without a mode, the chord ends once a valid combination has been pressed. However, the bug is that the chord does not end with an invalid key press.

Reply:
Is that a bug? Sounds like a feature!

Thanks for the clarification -- if I was right then the docs are clear enough :D